platform/spec/services/etl/extractors/workflow_execution_params_service_spec.rb

```
@@ -0,0 +1,150 @@
+
+        expect { service }.to raise_error(Etl::UnsupportedIntegrationType)
+      end
+    end
+
+    context "when an error occurs" do
+      let(:source) { create(:connector, integration_type: :api) }
+      let(:connection) { create(:connection, source: source) }
+      let(:sync) { create(:sync, connection: connection) }
+
+      before do
+        allow_any_instance_of(Etl::Extractors::ConnectorWorkflows::ApiStrategy)
```

rubocop (1.68.0)

[convention] Avoid stubbing using `allow_any_instance_of`.

(RSpec/AnyInstance)